### DISCUSSION TITLE
More ATAPI CD-ROM models?

### DISCUSSION THREAD

**Discussion by MaxwellS04** [86Box/86Box#4960] [Ideas]
I know there are 21 ATAPI CD-ROM models available (with 86BOX CD-ROM 1.00 having SCSI bus) on 86Box. Is there more ATAPI CD-ROM models to be added on 86Box?

**Comment by jriwanek** [comment]
There are none planned at this time to my knowledge, are there any specific models on your mind which are required for specific functionality, for example, passing a recovery disk's detection?
